src/Test/L1/Tasks/d9bafed4-0b18-4f58-968d-86655b4d2ce9/cmdline.js

```diff
@@ -0,0 +1,93 @@
+"use strict";
```

Comment:
Are these just copies of the tasks? Is there any way to get the current version instead of copying these? How will we maintain these copies?

Reply:
Its a copy of the cmdline task, though our model is extensible to allow any task to be added - I don't really see any need to get the current version or maintain the tasks. Our goal here isn't to test these tasks, its to test the agent. This is really just a way to run things like `##` commands while mocking as little as possible